### DISCUSSION TITLE
达梦数据库打包后仍报错。我用的是支持达梦的开发版。

### DISCUSSION THREAD

**Discussion by Bourne1314** [apache/incubator-seata#4187] [Q&A] [answered]
当我用达梦数据库时，A服务通过远程调用B服务报如下错误： 
Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.executor.ExecutorException: Error preparing statement.  Cause: io.seata.common.loader.EnhancedServiceNotFoundException: not found service provider for : io.seata.sqlparser.druid.SQLOperateRecognizerHolder] with root cause

o.seata.common.loader.EnhancedServiceNotFoundException: not found service provider for : io.seata.sqlparser.druid.SQLOperateRecognizerHolder
我用的是开发版本的，并且在rm-database这个包中有达梦的相关类和方法。
**_但我在打包之后，通过调用远程服务时，仍然报错：_**
**### not found service provider for : io.seata.sqlparser.druid.SQLOperateRecognizerHolder。**

**Comment by funky-eyes** [comment, marked as answer]
请补充完整堆栈

**Reply by Bourne1314** [reply]
已解决，是依赖的问题，多谢。
